solr/core/src/java/org/apache/solr/search/join/ScoreModeParser.java

```diff
@@ -29,7 +29,7 @@ class ScoreModeParser {
   private ScoreModeParser() {}
 
   private static Map<String, ScoreMode> getLowerAndCapitalCaseMap() {
-    Map<String, ScoreMode> map = new HashMap<>(ScoreMode.values().length * 2);
+    Map<String, ScoreMode> map = CollectionUtil.newHashMap(ScoreMode.values().length * 2);
```

Comment:
I suspect here it should be newHashSet(ScoreMode.values().length) (the author doubled, knowing he/she was setting a map capacity)